js/main.js

```diff
@@ -133,42 +164,50 @@ function printDigitalTime() {
 //    printOnScreen(translations[wordNumber], 180, 110, 20);
 //}
 
-window.onload = function() {
-    'use strict';
-    var wordNumber = 0;
-    
-    printDigitalTime();
-    
-    canvasWords = document.getElementById("wordSpace");
-    ctxWords = canvasWords.getContext('2d');
-    
-    document.getElementById("nextButton").addEventListener("click", function(){
-        wordNumber++;
-        if (wordNumber > numberOfWords) {
-        	wordNumber = 0;
-        }
-    	ctxWords.clearRect(0,0, 380,380);
-    	printOnScreen(words[wordNumber], 180, 70, 40);
-        
-    });
-    
-    document.getElementById("revealButton").addEventListener("click", function(){
-    	ctxWords.clearRect(0,0, 380,380);
-    	printOnScreen(words[wordNumber], 180, 70, 40);
-    	printOnScreen(translations[wordNumber], 180, 110, 20);
-    });
+function buttonEventListener() {
+	var wordNumber = 0;
+	
+	document.getElementById("nextButton").addEventListener("click", function(){
+	    wordNumber++;
+	    if (wordNumber > numberOfWords) {
+	    	wordNumber = 0;
+	    }
+		ctxWords.clearRect(0,0, SCREEN_WIDTH,SCREEN_HEIGHT);
+		printOnScreen(words[wordNumber], SCREEN_MIDDLE, WORD_POSY, MAX_TEXTSIZE);
+	    
+	});
+	
+	document.getElementById("revealButton").addEventListener("click", function(){
+		ctxWords.clearRect(0,0, SCREEN_WIDTH, WORDSPACE_HEIGHT);
+		printOnScreen(words[wordNumber], SCREEN_MIDDLE, WORD_POSY, MAX_TEXTSIZE);
+		printOnScreen(translations[wordNumber], SCREEN_MIDDLE, TRANSLATION_POSY, MIN_TEXTSIZE);
+	});
+}
 
-    // add eventListener for tizenhwkey
-    document.addEventListener('tizenhwkey', function(e) {
+function tizenBackButton() {
+	document.addEventListener('tizenhwkey', function(e) {
         if (e.keyName === "back") {
             try {
                 tizen.application.getCurrentApplication().exit();
             } catch (ignore) {}
         }
     });
+}
+
+function printFirstWord(){
```

Comment:
Maybe you should refactor this function to:
` +function printWord(word){
+	printOnScreen(word SCREEN_MIDDLE, WORD_POSY, MAX_TEXTSIZE);
+}`